### ISSUE TITLE
Is there anyway to distinguish which Gaussians are newly added after the desification?

### ISSUE THREAD

**Issue by lemonci** [VladimirYugay/MAGiC-SLAM#20]
Since in `mapper.Mapper.grow_submap`, you called [`gaussian_model.add_points(cloud_to_add)`](https://github.com/VladimirYugay/MAGiC-SLAM/blob/4ac5aa92ab70139546b42927b62e4f2fd6940222/src/entities/mapper.py#L166) and in `gaussian_model.add_points(cloud_to_add)` , the final step is [`self.densification_postfix`](https://github.com/VladimirYugay/MAGiC-SLAM/blob/4ac5aa92ab70139546b42927b62e4f2fd6940222/src/entities/gaussian_model.py#L154).

I wonder if after `self.densification_postfix`, if we wanna make a mask of the variable `global_points`  in the function `add_points` based on whether the points are newly added or no, is that possible?

Thanks.

**Comment by lemonci** [comment]
I guess directly save 

`xyz_points = self.get_xyz()
`
before 
`
global_points = torch.cat((xyz_points, pcd_points))`

Then

```
mask = torch.zeros(global_points.shape[0], dtype=torch.bool)
mask[:xyz_points.shape[0]] = True  
```
